Error with latest version 4.2.0

Please [read these guidelines](http://ibm.biz/cdt-issue-guide) before opening an issue.

<!-- Issues will be CLOSED IMMEDIATELY if the following template is not completed. -->

## Bug Description

### 1. Steps to reproduce and the simplest code sample possible to demonstrate the issue
Installing the latest version published a few hours ago i have now an error

```
  throw new Error(`Failed to load plugin - ${e.message}`);
            ^

Error: Failed to load plugin - Cannot find module '../plugins/cookieauth'
```

### 2. What you expected to happen

webpack compilation completes. Or maybe some instructions with breaking changes

thanks

### 3. What actually happened

a message about a missing plugin

## Environment details
<!--
- Version(s) that are affected by this issue.
    > 4.2.0
- Node.js version
    > 12.8.1
webpack 4.39.3
typescript 3.5.2
-->

full stack 
```
            throw new Error(`Failed to load plugin - ${e.message}`);
            ^

Error: Failed to load plugin - Cannot find module '../plugins/cookieauth'
    at /Users/guyhuinen/Documents/workspace/leap/leap/applications/expo/server/server.js:1047:19
    at Array.forEach (<anonymous>)
    at CloudantClient._addPlugins (/Users/guyhuinen/Documents/workspace/leap/leap/applications/expo/server/server.js:1007:13)
    at new CloudantClient (/Users/guyhuinen/Documents/workspace/leap/leap/applications/expo/server/server.js:997:10)
    at Cloudant (/Users/guyhuinen/Documents/workspace/leap/leap/applications/expo/server/server.js:298:24)
    at new cloudant_CloudantLib (/Users/guyhuinen/Documents/workspace/leap/leap/applications/expo/server/server.js:2606:43)
    at Module.fc4506377efd97041428 (/Users/guyhuinen/Documents/workspace/leap/leap/applications/expo/server/server.js:2616:27)
    at __webpack_require__ (/Users/guyhuinen/Documents/workspace/leap/leap/applications/expo/server/server.js:20:30)
    at /Users/guyhuinen/Documents/workspace/leap/leap/applications/expo/server/server.js:84:18
    at Object.<anonymous> (/Users/guyhuinen/Documents/workspace/leap/leap/applications/expo/server/server.js:87:10)
    at Module._compile (internal/modules/cjs/loader.js:868:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:879:10)
    at Module.load (internal/modules/cjs/loader.js:731:32)
    at Function.Module._load (internal/modules/cjs/loader.js:644:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:931:10)
    at internal/main/run_main_module.js:17:11
```

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,3 +1,6 @@
+# UNRELEASED
+- [FIXED] Include all built-in plugin modules in webpack bundle.
+
 # 4.2.0 (2019-08-27)
 - [NEW] Added option to set new IAM API key.
 - [FIXED] Allow plugins to be loaded from outside the 'plugins/' directory.

--- a/lib/client.js
+++ b/lib/client.js
@@ -100,12 +100,7 @@ class CloudantClient {
           }
 
           var pluginName = Object.keys(plugin)[0];
-
-          try {
-            Plugin = require(self._buildPluginPath(pluginName));
-          } catch (e) {
-            throw new Error(`Failed to load plugin - ${e.message}`);
-          }
+          Plugin = self._importPlugin(pluginName);
 
           cfg = plugin[pluginName];
           if (typeof cfg !== 'object' || Array.isArray(cfg)) {
@@ -119,12 +114,7 @@ class CloudantClient {
             return; // noop
           }
 
-          try {
-            Plugin = require(self._buildPluginPath(plugin));
-          } catch (e) {
-            throw new Error(`Failed to load plugin - ${e.message}`);
-          }
-
+          Plugin = self._importPlugin(plugin);
           cfg = {};
           break;
 
@@ -161,6 +151,30 @@ class CloudantClient {
 
     // A relative path was provided
     return path.join(process.cwd(), name);
+  }
+
+  _importPlugin(pluginName) {
+    switch (pluginName) {
+      // Note: All built-in plugins are individually listed here to ensure they
+      //       are included in a webpack bundle.
+      case 'cookieauth':
+        return require('../plugins/cookieauth');
+      case 'iamauth':
+        return require('../plugins/iamauth');
+      case 'retry':
+        return require('../plugins/retry');
+      default:
+        // Warning: Custom plugins will not be included in a webpack bundle
+        //          by default because the exact module is not known on compile
+        //          time.
+        try {
+          // Use template literal to suppress 'dependency is an expression'
+          // webpack compilation warning.
+          return require(`${this._buildPluginPath(pluginName)}`);
+        } catch (e) {
+          throw new Error(`Failed to load plugin - ${e.message}`);
+        }
+    }
   }
 
   _initClient(client) {

